homeassistant/core.py

```diff
@@ -426,6 +428,12 @@ def stop(self) -> None:
         await self.async_block_till_done()
 
         # stage 2
+        self.state = CoreState.writing_data
+        self.async_track_tasks()
+        self.bus.async_fire(EVENT_HOMEASSISTANT_WRITE_DATA)
```

Comment:
This name should be improved. Right now it's too generic. Maybe something like `EVENT_HOMEASSISTANT_FINAL_WRITE` ?

Reply:
(same with the value)

Reply:
done